### PR TITLE
[docs] update latest benchmark results

### PR DESCRIPTION
**Pull request by guolinke**

### CODE REVIEW

File: docs/Experiments.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ Comparison Experiment
 
 For the detailed experiment scripts and output logs, please refer to this `repo`_.
 
+History
+^^^^^^^
+08 Mar, 2020: update according to the latest master barnch. (``xgboost_extra`` is not updated for it is too slow.)
```
Comment (guolinke) [inline docs/Experiments.rst]:
```suggestion
08 Mar, 2020: update according to the latest master branch. (``xgboost_extra`` is not updated for it is too slow.)
```

**Comment by Laurae2** [comment]
@guolinke Which commits of xgboost / LightGBM did you use for the benchmarks?

**Comment by StrikerRUS** [comment]
@guolinke Very impressive!
What about `Parallel Experiment`? I think it should be updated as well or be marked as from `first version`.

**Comment by guolinke** [comment]
@StrikerRUS as we didn't improve parallel algorithm, we can mark them to `first version`

**Pull request merged**